src/main/java/de/dhbw/karlsruhe/students/mailflow/App.java

```diff
@@ -56,7 +57,8 @@ public static void main(String[] args) {
         UCCollectionSearchEmail.init(provideEmails.provideAllEmailsService());
     final UCCollectionSettings collectionSettings =
         UCCollectionSettings.init(authSession, userSettingsRepository,
-            new ScheduledSendTimeParserService());
+            new ScheduledSendTimeParserService(),
+            new ChangePasswordService(userRepository, authSession));
```

Comment:
please create these two classes in the init method. Otherwise this init method makes no sense.

Reply:
well I was also wondering why there is this `init()` method... we're completely losing the flexibility to change everything there? Can't we just get rid of the `init()` method?

Reply:
and, if I remember correctly, the `new ScheduledSendTimeParserService()` was your suggestion from the other PR... could you please clarify what was the intention here?

Reply:
The init method is there so that this main method looks more compact. And until now we did not loose any flexibility to change because its basically the same as you did here. I thought it just looks cleaner this way because we had so many services that this `main` was hard to read.

Reply:
I see, thanks for the clarification! :)